src/main/java/stsjorbsmod/cards/wanderer/Mania.java

```diff
@@ -50,6 +50,42 @@ private boolean isEligibleForExtraEffect() {
         return UniqueCardUtils.countUniqueCards(AbstractDungeon.player.hand) == AbstractDungeon.player.hand.size();
     }
 
+    @Override
+    public void applyPowers() {
+        int count = calculateBonusBaseDamage();
+        if(count>0) {
+            super.applyPowers();
+            if(this.upgraded) {
+                this.rawDescription = cardStrings.UPGRADE_DESCRIPTION + cardStrings.EXTENDED_DESCRIPTION[0];
+            }
+            else
+                this.rawDescription = cardStrings.DESCRIPTION + cardStrings.EXTENDED_DESCRIPTION[0];
+            initializeDescription();
+        }
+    }
+
+    @Override
+    public void onMoveToDiscardImpl() {
+        if(this.upgraded) {
+            this.rawDescription = cardStrings.UPGRADE_DESCRIPTION;
+        }
+        else
+            this.rawDescription = cardStrings.DESCRIPTION;
+        initializeDescription();
+    }
+
+    @Override
+    public void calculateCardDamage(AbstractMonster mo) {
+        super.calculateCardDamage(mo);
+        if(this.upgraded) {
```

Comment:
For the sake of localization (where there may be cases where some languages require UPGRADE_DESCRIPTIONs even when english does not because of different pluralization rules/etc), I think it would be a good idea for all of the different cards with this sort of logic to include this "if upgraded" check even if it isn't necessary in english for all the cards.

Maybe using helpers in `CustomJorbsModCard` would help reduce the boilerplate for doing this? Something like

```java
// CustomJorbsModCard
protected String rawDynamicDescriptionSuffix = "";

@Override
public void initializeDescription() {
    String originalRawDescription = this.rawDescription;
    this.rawDescription += this.rawDynamicDescriptionSuffix;
    super.initializeDescription();
    this.rawDescription = originalRawDescription;
}

// Individual cards...
@Override
public void applyPowers() {
    super.applyPowers();
    this.rawDynamicDescriptionSuffix = cardStrings.EXTENDED_DESCRIPTION[0];
}
@Override
public void calculateCardDamage(AbstractMonster mo) {
    super.calculateCardDamage(mo);
    this.rawDynamicDescriptionSuffix = cardStrings.EXTENDED_DESCRIPTION[0];
}
@Override
public void onMoveToDiscardImpl() {
    this.rawDynamicDescriptionSuffix = "";
}
```